docs/source/private_data_tutorial.rst

```diff
@@ -763,37 +791,9 @@ to this, lets continue.
 
 :guilabel:`Try it yourself`
 
-Since we need to keep track of how many blocks we add to see if our private data gets purged, open a new terminal window and view the private data logs for this peer by
-running the following command. Note the highest block number.
-
-.. code:: bash
-
-    docker logs peer0.org1.example.com 2>&1 | grep -i -a -E 'private|pvt|privdata'
-
-
-Switch back to the terminal with our peer CLI. Now that we are operating
-as a member of Org2, we can demonstrate that the asset appraisal is not stored
-in Org2MSPPrivateCollection, on the Org2 peer:
-
-.. code:: bash
-
-    peer chaincode query -o localhost:7050 --ordererTLSHostnameOverride orderer.example.com --tls --cafile ${PWD}/organizations/ordererOrganizations/example.com/orderers/orderer.example.com/msp/tlscacerts/tlsca.example.com-cert.pem -C mychannel -n private -c '{"function":"ReadAssetPrivateDetails","Args":["Org2MSPPrivateCollection","asset1"]}'
-
-The empty response shows that the asset1 private details do not exist in buyer (Org2) private collection.
```

Comment:
This queries were repetitive with the section above, so I consolidated them and removed the queries in this section.